app/api/library-hours/route.tsx

```diff
@@ -2,6 +2,7 @@ import {deserialize} from "@/lib/drupal/deserialize"
 import {NextResponse} from "next/server"
 import {DayHours} from "@/lib/hooks/useLibraryHours"
 import {LibraryHours} from "@/lib/drupal/drupal"
+import {unstable_cache as nextCache} from "next/cache"
```

Comment:
I'm curious why you're aliasing this here?

Reply:
`unstable_cache()` just didn't look very pretty to me :smile:

Reply:
Fair enough.  Makes no difference in the functionality, I was just curious.